lib/logging.go

```diff
@@ -0,0 +1,21 @@
+package lib
+
+import (
+	"github.com/op/go-logging"
+	"os"
+)
+
+// Init logger for the lib package
+var logger = GetLogger()
+
+// GetLogger Will initialize a logger
```

Comment:
Since this is an open source project it might be that at some point someone with not so sophisticated go knowlegde looks at this and think that could be removed ;-)
The comment does not hurt

Reply:
Yep, it does hurt. Though current wording also makes no point as it states obvious.
Probably — for the sake of guardrailing it — it makes sense to add comment like `Logger initialization required for downstream dependant functions`?

Reply:
Will fix that. Thanks for the input